lua/net.lua

```diff
@@ -1,10 +1,14 @@
 local gatekeeperc = require("gatekeeperc")
+local ffi = require("ffi")
 
 local M = {}
 
 -- Function that sets up the network.
 function M.setup_block()
 
+	local front_ports = {"0000:83:00.0"}
+	local back_ports = {"0000:85:00.0"}
```

Comment:
That sounds much cleaner, thanks. Any preference on where this new file should go? For example, we could place it in `/etc/if_map.lua` so that all users can access it or could put it in the gatekeeper repository itself.